filnet/router.go

```diff
@@ -39,3 +42,12 @@ func NewRouter(r routing.IpfsRouting) *Router {
 func (r *Router) FindProvidersAsync(ctx context.Context, key cid.Cid, count int) <-chan pstore.PeerInfo {
 	return r.routing.FindProvidersAsync(ctx, key, count)
 }
+
+// GetClosestPeers returns peers of the K closest peers to the given key
```

Comment:
To help make it clear what K is please modify to something like "GetClosestPeers returns a channel of the K closest peers to the given key where K is the 'K Bucket' parameter of the Kademlia DHT protocol.

Also please be sure to put a fullstop at the end of a comment like this that is a complete sentence as some people here are a little OCD about that 😀.